src/index.html

```diff
@@ -7,6 +7,130 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
```suggestion
    <div class="calendar calendar--start-day-mon calendar--month-length-31">
```
It's enough to have modifiers only in the parent class